kinto_http/__init__.py

```diff
@@ -259,6 +267,89 @@ def delete_buckets(self, safe=True, if_match=None, if_exists=False):
         resp, _ = self.session.request('delete', endpoint, headers=headers)
         return resp['data']
 
+    # Groups
+
+    def get_groups(self, bucket=None):
+        endpoint = self.get_endpoint('groups', bucket=bucket)
+        return self._paginated(endpoint)
+
+    def create_group(self, group=None, bucket=None,
```

Comment:
What happens if `group` is left `None` here?

We could do a `POST` on the plural endpoint instead of `PUT` and let the server generate a group name for us.